### ISSUE TITLE
Incomplete Nation Profiles

### ISSUE THREAD

**Issue by xsttx** [Bit-Nation/bitnation-web#40]
I think we need to force people to respond to _all _ questions when creating a nation, so people who are not serious about it don't do it. Now it looks like a lot are registering simply to test it, not because they're serious about starting a nation.

**Comment by evok3d** [comment]
Good idea. Do you want all or just some of the most important ones to be mandatory? Perhaps the key ones and other ones can be left as optional?

**Comment by xsttx** [comment]
I think all of them apart from superficial ones like flag, national anthem, logo etc.
